frontend/public/components/secrets/create-secret.tsx

```diff
@@ -101,6 +101,24 @@ const generateSecret = () => {
   return s4() + s4() + s4() + s4();
 };
 
+const determineSecretType = (stringData): SecretType => {
```

Comment:
`stringData` makes it sounds like the parameter is a string, but it appears to be an object.

Reply:
@spadgett so the `stringData` is name of the [field](https://kubernetes.io/docs/concepts/configuration/secret/#overview-of-secrets). Similar to `data` but this take an arbitrary string value.